bundles/org.openhab.binding.openthermgateway/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -322,38 +342,378 @@
 	</channel-type>
 
 	<channel-type id="burnerhours">
-		<item-type>Number:Time</item-type>
+		<item-type>Number:Dimensionless</item-type>
 		<label>Burner Hours</label>
 		<description>Burner hours</description>
 		<state readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="chpumphours">
-		<item-type>Number:Time</item-type>
+		<item-type>Number:Dimensionless</item-type>
 		<label>Central Heating Pump Hours</label>
 		<description>Central heating pump hours</description>
 		<state readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="dhwpvhours">
-		<item-type>Number:Time</item-type>
+		<item-type>Number:Dimensionless</item-type>
 		<label>Domestic Hot Water Pump/Valve Hours</label>
 		<description>Domestic hot water pump/valve hours</description>
 		<state readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="dhwburnerhours">
-		<item-type>Number:Time</item-type>
+		<item-type>Number:Dimensionless</item-type>
 		<label>Domestic Hot Water Burner Hours</label>
 		<description>Domestic hot water burner hours</description>
 		<state readOnly="true"/>
 	</channel-type>
 
-	<channel-type id="sendcommand">
-		<item-type>String</item-type>
-		<label>Send Command</label>
-		<description>Channel to send commands to the OpenTherm Gateway device</description>
-		<state readOnly="false"/>
+	<channel-type id="tspnumber">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Transparent Slave Parameter Number</label>
+		<description>Number of transparant slave parameter entries</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="tspentry">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Transparent Slave Parameter Entry</label>
+		<description>Transparent slave parameter entry</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="fhbnumber">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Fault History Buffer Number</label>
+		<description>Number of fault history buffer entries</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="fhbentry">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Fault History Buffer Entry</label>
+		<description>Fault history buffer entry</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<!-- ventilationheatrecovery -->
+
+	<channel-type id="vh_ventilationenable">
+		<item-type>Switch</item-type>
+		<label>Ventilation Enabled</label>
```

Comment:
```suggestion
		<label>Ventilation</label>
```

Reply:
I think this is in the same line as your comment about using a prefix? In that case, please read my reply on that.

Reply:
My comment has nothing to do with the prefix in the Channel UID.

"Enabled" is redundant as the switch shows if Ventilation is ON or OFF. However, I won't insist on changing it.

Reply:
IMHO `ENABLED` is quite different than `ON` .. the former means that it is allowed to be either on or off; whereas the latter means that it is definitely off..